tests/scene/test_path_follow_3d.h

```diff
@@ -283,4 +283,54 @@ TEST_CASE("[SceneTree][PathFollow3D] Calculate forward vector") {
 
 	memdelete(path);
 }
+
+TEST_CASE("[SceneTree][PathFollow3D] Calculate forward vector with degenerate curves") {
+	Ref<Curve3D> curve = memnew(Curve3D);
+
+
+	curve->add_point(Vector3(0, 0, 1), Vector3(), Vector3(1, 0, 0));
```

Comment:
Two consecutive empty lines fails the style checks:
```suggestion
	Ref<Curve3D> curve = memnew(Curve3D);

	curve->add_point(Vector3(0, 0, 1), Vector3(), Vector3(1, 0, 0));
```